### DISCUSSION TITLE
Add `version_url_template` in the theme schema

### DISCUSSION THREAD

**Discussion by lewis-yeung** [JanDeDobbeleer/oh-my-posh#2274] [Ideas]
Should we add `properties -> version_url_template` (which corresponds to `VersionURLTemplate`) for those language segments that support it to the theme schema? I notice that currently it is only mentioned in docs of a `dotnet` segment.

**Comment by JanDeDobbeleer** [comment]
We could :-)

**Reply by lewis-yeung** [reply]
Okay, will prepare a PR.
